allenact/base_abstractions/experiment_config.py

```diff
@@ -70,6 +71,17 @@ def __init__(
         self._sensor_preprocessor_graph_cached: Optional[SensorPreprocessorGraph] = None
         self._visualizer_cached: Optional[VizSuite] = None
 
+        self.local_worker_ids: Optional[List[int]] = None
+        self.set_local_worker_ids(local_worker_ids or list(range(len(self.devices))))
+
+    def set_local_worker_ids(self, local_worker_ids: List[int]):
+        self.local_worker_ids = local_worker_ids
+
+        assert all(0 <= id < len(self.devices) for id in self.local_worker_ids), (
+            f"Passed {len(self.local_worker_ids)} local worker ids {self.local_worker_ids} "
+            f"for {len(self.devices)} total devices (workers)"
```

Comment:
A style tip I learned from Nicholas Lourie: when doing multiple line strings like this it can be helpful to always start new lines with a space as it makes it easy to scan all the lines and make sure you haven't forgotten to put one. I.e. it's easier to see a space is missing in
```python
(
    "A style tip I learned from Nicholas"
    " Lourie: when doing multiple line strings like this"
    " it can be helpful to always start new lines with a"
    "space as it makes it easy to scan all the lines and make"
    " sure you haven't forgotten to put one. I.e. it's easier"
    " to see a space is missing in"
)
```
than in
```python
(
    "A style tip I learned from Nicholas "
    "Lourie: when doing multiple line strings like this "
    "it can be helpful to always start new lines with a"
    "space as it makes it easy to scan all the lines and make "
    "sure you haven't forgotten to put one. I.e. it's easier "
    "to see a space is missing in "
)
```

Reply:
Good tip! 👍